boards/arm/wio_terminal/doc/index.rst

```diff
@@ -0,0 +1,199 @@
+display, USB C port, FPC connector, MicroSD card slot, Raspberry pi compatible
+40pin header and 2 Grove connectors.
+
+.. image:: img/wio_terminal.png
+     :width: 500px
+     :align: center
+     :alt: Seeed Studio Wio Terminal
+
+Hardware
+********
+
+- ATSAMD51P19 ARM Cortex-M4F processor at 120 MHz (boost up to 200 MHz)
+- 512 KiB flash memory and 192 KiB of RAM
+- 4 MiB external flash
+- MicroSD card slot
+- RTL8720DN 2.4G/5G Dual Bands Wireless and BLE5.0 Combo Module
+- 2.4inch LCD display
+- LIS3DH accelerometer
+- Microphone 1.0V-10V -42dB
+- Speaker ≥78dB @10cm 4000Hz
+- Light Sensor 400-1050nm
+- Infrared Emitter 940nm
+- GPIO 40 pin (Raspberry Pi compatible)
+- 2x Grove connectors
+- 1x user LED
+- 3x user buttons
+- 5-way user button
+- Power/Reset/Boot mode switch
+- Native USB port
+
+Supported Features
+==================
+
+The wio_terminal board configuration supports the following
+hardware features:
+
++-----------+------------+------------------------------------------+
+| Interface | Controller | Driver/Component                         |
++===========+============+==========================================+
+| NVIC      | on-chip    | Nested vector interrupt controller       |
++-----------+------------+------------------------------------------+
+| Flash     | on-chip    | Can be used with LittleFS to store files |
++-----------+------------+------------------------------------------+
+| SYSTICK   | on-chip    | Systick                                  |
++-----------+------------+------------------------------------------+
+| WDT       | on-chip    | Watchdog                                 |
++-----------+------------+------------------------------------------+
+| GPIO      | on-chip    | I/O ports                                |
++-----------+------------+------------------------------------------+
+| USART     | on-chip    | Serial port                              |
++-----------+------------+------------------------------------------+
+| I2C       | on-chip    | Inter-Integrated Circuit                 |
++-----------+------------+------------------------------------------+
+| SPI       | on-chip    | Serial Peripheral Interface port         |
++-----------+------------+------------------------------------------+
+| TRNG      | on-chip    | True Random Number Generator             |
++-----------+------------+------------------------------------------+
+| HWINFO    | on-chip    | Unique 128 bit serial number             |
++-----------+------------+------------------------------------------+
+| RTC       | on-chip    | Real-Time Counter                        |
++-----------+------------+------------------------------------------+
+| USB       | on-chip    | USB device                               |
++-----------+------------+------------------------------------------+
+| PWM       | on-chip    | PWM                                      |
++-----------+------------+------------------------------------------+
```

Comment:
Update table to new format, see [atsamr34_xpro](https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/atsamr34_xpro/doc/index.rst) as an example.

Reply:
Done